android_automation/page_action/category_page.py

```diff
@@ -205,3 +206,31 @@ def click_category_product(wd, text):
     aalc(wd, f'c_{text}')
     sleep(1)
     navigation_bar.close_bottom_sheet(wd)
+
+
+def click_first_large_category(wd):
+    category_layer = aal(wd, 'com.the29cm.app29cm:id/shopComposeView')
+    # 첫번째 대메뉴 선택
+    aalc(category_layer, 'category_first_title')
+
+
+def check_unique_medium_category(self, wd):
+    # 중 카테고리 리스트 중 상단 4개의 카테고리명을 리스트로 저장
+    medium_category_list = []
+    category_layer = aal(wd, 'com.the29cm.app29cm:id/shopComposeView')
+
+    medium_category_list.append(aal(category_layer,
+                                    '//android.view.View/android.view.View[3]/android.view.View[@index=3]/android.widget.TextView').text)
+    medium_category_list.append(aal(category_layer,
+                                    '//android.view.View/android.view.View[3]/android.view.View[@index=4]/android.widget.TextView').text)
+    medium_category_list.append(aal(category_layer,
+                                    '//android.view.View/android.view.View[3]/android.view.View[@index=5]/android.widget.TextView').text)
+    medium_category_list.append(aal(category_layer,
+                                    '//android.view.View/android.view.View[3]/android.view.View[@index=6]/android.widget.TextView').text)
```

Comment:
append 할 요소들의 index 값만 변경된다면 for문으로 작성하는 것이 좀 더 가독성이 좋지 않을까요?? 🤔 
```suggestion
for i in range(3, 7):
    medium_category_list.append(aal(category_layer,
                                f'//android.view.View/android.view.View[3]/android.view.View[@index={i}]/android.widget.TextView').text)
```

Reply:
넵 path 더 줄일 수 있는 방법 찾아보려고 하는데 일단은 다정님이 제안주신 코드로 수정했습니다.